components/camel-graphql/src/main/java/org/apache/camel/component/graphql/GraphqlEndpoint.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.camel.component.graphql;
+
+import org.apache.camel.Component;
+import org.apache.camel.Consumer;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.support.DefaultEndpoint;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
```

Comment:
Here we are missing `@UriEndpoint` annotation with the schema information, you can see other compenents for some examples

Reply:
Added the annotation

Reply:
Done